setup.py

```diff
@@ -129,41 +128,36 @@ def py2exe_options():
     author_email=author_email,
     classifiers=filter(None, classifiers.split('\n')),
     license=license,
-    scripts=['openerp-server', 'odoo.py'],
+    entry_points={
+        'console_scripts': [
+            'odoo=odoo.cli:main',
+        ]
+    },
     packages=find_packages(),
-    package_dir={'%s' % lib_name: 'openerp'},
+    package_dir={'%s' % lib_name: 'odoo'},
     include_package_data=True,
     install_requires=[
         'babel >= 1.0',
         'decorator',
         'docutils',
         'feedparser',
-        'gevent',
         'Jinja2',
         'lxml',  # windows binary http://www.lfd.uci.edu/~gohlke/pythonlibs/
         'mako',
         'mock',
-        'ofxparse',
         'passlib',
         'pillow',  # windows binary http://www.lfd.uci.edu/~gohlke/pythonlibs/
         'psutil',  # windows binary code.google.com/p/psutil/downloads/list
-        'psycogreen',
```

Comment:
@sle-odoo ok, now I read your commit message. These are considered optional dependencies.

Perhaps can you declare them using setuptools extra_requires?
http://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies